src/main/webapp/scheduled-interviews.jsp

```diff
@@ -23,6 +23,16 @@
             <li class="list-group-item">Interviewee: ${scheduledInterview.getInterviewee()}</li>
             <li class="list-group-item">Interviewer: ${scheduledInterview.getInterviewer()}</li>
           </ul>
+          <c:choose>
+            <c:when test= "${scheduledInterview.getHasPassed()}">
+              <a href="feedback.html?interview=${scheduledInterview.getId()}&role=${scheduledInterview.getRole()}" style="text-align:center">
+                <button class = "feedbackButton">Submit Feedback</button>
+              </a>
+            </c:when>
+            <c:otherwise>
+              <p style="text-align:center">You may submit feedback once the interview has commenced.</p>
```

Comment:
Commenced l feel is so fancy, l had to look it up. Kudos on the good vocab but dumb it down please to "started" ?
Also just to be accurate change to 5 min before it starts

Reply:
Done